view/bindings/bindings.js

```diff
@@ -176,26 +176,87 @@ steal("can/util", "can/view/callbacks", "can/control", function (can) {
 			// 
 			// For example, can-submit binds on the submit event.
 			event = attributeName.substr("can-".length),
-			// This is the method that the event will initially trigger. It will look up the method by the string name 
+			// This is the method that the event will initially trigger. It will look up the method by the string name
 			// passed in the attribute and call it.
 			handler = function (ev) {
-				// The attribute value, representing the name of the method to call (i.e. can-submit="foo" foo is the 
-				// name of the method)
 				var attrVal = el.getAttribute(attributeName);
-				// if the attribute is not present currently, don't run the event handler, but don't unbind,  
-				// since it might just be temporarily hidden
-				if(!attrVal){
-					return false;
+				if (!attrVal) { return; }
+				// mustacheCore.expressionData will read the attribute
+				// value and parse it identically to how mustache helpers
+				// get parsed.
+				var attrInfo = mustacheCore.expressionData(removeCurly(attrVal));
+
+				// We grab the first item and treat it as a method that
+				// we'll call.
+				var scopeData = data.scope.read(attrInfo.name.get, {
+					returnObserveMethods: true,
+					isArgument: true
+				});
+
+				// We break out early if the first argument isn't available
+				// anywhere.
+				
+				//!steal-remove-start
+				if (!scopeData.value) {
+					can.dev.warn("can/view/bindings: " + attributeName + " couldn't find method named " + attrInfo.name.get, {
+						element: el,
+						scope: data.scope
+					});
+					return null;
 				}
+				//!steal-remove-end
+
+				var args = [];
+				var $el = can.$(this);
+				var localScope = data.scope.add({
+					"@element": $el,
+					"@event": ev,
+					"@scope": can.scope($el[0]),
```

Comment:
@zkat or @daffl Can we remove this line for now, until #1300 is figured out. I'm pretty sure I'll want @scope to mean the actual `can.view.Scope` at that time.